dvc/command/metrics.py

```diff
@@ -98,7 +99,9 @@ def add_parser(subparsers, parent_parser):
         description=METRICS_SHOW_HELP,
         help=METRICS_SHOW_HELP,
     )
-    metrics_show_parser.add_argument("path", nargs="?", help="Path to metrics file.")
+    metrics_show_parser.add_argument(
+        "path", nargs="?", help="Path to metrics file or folder"
```

Comment:
Let's use `directory` instead of `folder` here and down below since we use it in every other place.